src/NServiceBus.Core/Transports/TransportInfrastructure.cs

```diff
@@ -65,5 +66,21 @@ public virtual string MakeCanonicalForm(string transportAddress)
         {
             return transportAddress;
         }
+
+        /// <summary>
+        /// Performs any action required to warm up the transport infrastructure before starting an edpoint.
+        /// </summary>
+        public virtual Task Start()
+        {
+            return TaskEx.CompletedTask;
+        }
+
+        /// <summary>
+        /// Performs any action required to cool down the transport infrastructure after stopping an endpoint.
```

Comment:
Is "after stopping" correct? 

Isn't it more like "Allows the transport to perform cool down actions when the endpoint is stopping"?